jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -65,6 +65,30 @@ <h4 class="bold-title">
                 </div>
             </div>
         </section>
+        <section class="each-step-wrap api-deployment" *ngIf="typeOfService === 'api'">
+        <div class="row">
+          <div class="col-lg-4 col-md-4">
+            <p class="step-title">Choose your API deployment target<span>*</span></p>
+          </div>
+          <div class="col-lg-8 col-md-8">
+            <section class="col-lg-3 col-md-3 pad-left0">
+              <div class="radio-container" (click)="apiDeployment='aws_apigateway'">
+                <input type="radio" name="target" value="aws_apigateway" [ngModel]="apiDeployment">
+                <label><span class='outer'></span><span class="background"></span>API Gateway</label>
+              </div>
+            </section>
+            <section class="col-lg-3 col-md-3 pad-left0">
+              <div class="radio-container" (click)="apiDeployment='gcp_apigee'">
```

Comment:
Upon QA, I have seen that the previous selection (ex: apigee) is not getting reset to the default (aws api gateway) after the service got created successfully. However, when I refresh the page, default is getting assigned as expected.